scripts/reborn_webui_v2_live_qa/slack_helpers.py

```
@@ -143,21 +151,15 @@ def _configure_slack_legacy_actor_if_needed(
         ).strip()
     )
     if not slack_user_id:
-        return False, None
-    changed = _set_slack_section_key(config_path, "slack_user_id", slack_user_id)
-    return changed, slack_user_id
+        return None
+    return slack_user_id
 
 
 def _discover_slack_dm_route_channel(
     config_text: str,
```

Ruff (0.15.20)

[warning] Unused function argument: `config_text`

(ARG001)